.git-blame-ignore-revs

```diff
@@ -0,0 +1,12 @@
+# Ignored commits for git blame reccomendation
+
+# run `git config blame.ignoreRevsFile .git-blame-ignore-revs` to ignore 
+# the following autogenerated commits
+
+# frontend: `eslint` change identation to two spaces
+
+705423d833830d0522d9eda10f704271706e8d7e
+
+# frontend: eslint change switchcase identation
+
+0df2064cce37b26aa41061e1bd931500c71b73b1
```

Comment:
not sure about this. One concern about different commits was with `git bisect`, when those 2 commits aren't building, bisect doesn't work.

maybe better just squash the 3 commits into 1 ?

cc @x1ddos

Reply:
Those two commits work, this is simply an ignore rule for blame, not bisect